src/content/learn/writing-markup-with-jsx.md

```diff
@@ -308,7 +311,7 @@ export default function Bio() {
 
 </Sandpack>
 
-Whether to do it by hand or using the converter is up to you!
+Kamu bebas untuk mengubah secara manual atau menggunakan *converter*!
```

Comment:
```suggestion
Apakah melakukan dengan manual atau menggunakan *converter*, terserah pada Anda!
```

Reply:
Another suggestion to have better reading flow:

```suggestion
Antara melakukannya dengan manual atau menggunakan *converter*, terserah Anda!
```